src/archive/mod.rs

```diff
@@ -343,6 +374,14 @@ pub struct Archive<'a> {
 
 impl<'a> Archive<'a> {
     pub fn parse(buffer: &'a [u8]) -> Result<Archive<'a>> {
+        #[derive(PartialEq)]
+        enum IndexType {
+            NoIndex,
+            SysV,
+            WindowsLinkerMember,
```

Comment:
Seems a bit verbose, would simply `Windows` suffice?

Reply:
Sounds good to me.